tests/test_wcs.py

```diff
@@ -2488,7 +2488,12 @@ def test_fittedsipwcs():
 
     dir = 'fits_files'
 
-    nref = 100
+    if __name__ != "__main__":
+        # For pytest runs, don't bother with the crazy ones.  We really only care about
+        # testing that the code works correctly.  That can be done with just a couple of these.
+        test_tags = ['TAN', 'TPV', 'SIP', 'TNX']
+
+    nref = 300  # Use more than 256, since that's the block length in C++.
```

Comment:
Note: I wanted to exercise the n>256 version in the C++ code, so I bumped nref to 300.  This was useful, since it found a bug initially in my looping there.  But once I got things working, some of the new objects failed the old tolerances.  So that's why some of those numbers above changed.  I don't think this is pointing to any bug.  Just that there are more objects being extrapolated a little farther from where the constraints are.